src/__tests__/browserSuites/push-synchronization-retries.spec.js

```diff
@@ -54,11 +54,11 @@ const MILLIS_DESTROY = 700;
  *  0.2 secs: SPLIT_UPDATE event -> /splitChanges: bad response and then success -> SDK_UPDATE triggered
  *  0.3 secs: SPLIT_UPDATE event with old changeNumber -> SDK_UPDATE not triggered
  *  0.4 secs: MY_SEGMENTS_UPDATE event -> /mySegments/nicolas@split.io: fail and then success -> SDK_UPDATE triggered
- *  0.5 secs: SPLIT_KILL event -> /splitChanges: bad response and network error -> SDK_UPDATE not triggered although cache is updated (killLocally) @TODO review if we should trigger SDK_UPDATE anyway
+ *  0.5 secs: SPLIT_KILL event -> /splitChanges: bad response and network error -> SDK_UPDATE triggered although fetches fail
  *  0.6 secs: MY_SEGMENTS_UPDATE event -> /mySegments/nicolas@split.io: fail twice -> SDK_UPDATE not triggered
  */
 export function testSynchronizationRetries(mock, assert) {
-  assert.plan(14);
+  assert.plan(16);
```

Comment:
(If you can add a small comment as of why 16, that's great)

Reply:
this file was moved to PR #397 (comment will be added there)